front/src/components/student-registration/student-registration-page/student-registration-page.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import obj from '../../../common/redux/actions';
+import { connect } from 'react-redux';
+import { Modal, Row, Col } from 'antd';
+import StudentRegistrationForm from '../student-registration-form';
+
+class StudentRegistrationPage extends Component {
+  constructor(props) {
+    super(props);
+    this.handleHideModal = this.handleHideModal.bind(this);
+  }
+
+  handleHideModal() {
+    this.props.dispatch(obj.hideStudentRegisteredModal());
+  }
+
+  render() {
+    const { errorMessage, isRegistered, isModalVisible } = this.props;
+
+    if (!isRegistered) {
+      return (
+        <>
+          <StudentRegistrationForm
+            token={this.props.match.params.token}
+            dispatch={this.props.dispatch}
```

Comment:
You already use destructuring assignment of `this.props`. I think would be better to add `params` and `dispatch` there as well

Reply:
Yes, I think it will look a little bit better.